browser/src/GenePage/MitochondrialRegionConstraintTrack.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Exon } from '../TranscriptPage/TranscriptPage'
+import ConstraintTrack, { regionsInExons, RegionAttributeList } from '../ConstraintTrack'
+
+export type MitochondrialConstraintRegion = {
+  start: number
+  stop: number
+  oe: number
+  oe_upper: number
+  oe_lower: number
+}
+
+type Props = {
+  constraintRegions: MitochondrialConstraintRegion[] | null
+  exons: Exon[]
+}
+
+const constraintColor = '#fd8d3c'
+
+const Legend = () => (
+  <>
+    <span>Constrained region</span>
+    <svg width={50} height={25}>
+      <rect x={10} y={3} width={30} height={10} stroke="#000" fill={constraintColor} />
+    </svg>
+  </>
+)
+
+type TooltipProps = {
+  region: MitochondrialConstraintRegion
+}
+
+const Tooltip = ({ region }: TooltipProps) => {
+  return (
+    <RegionAttributeList>
+      <div>
+        <dt>Coordinates:</dt>
+        <dd>{`m:${region.start}-${region.stop}`}</dd>
```

Comment:
This is very minor -- it appears as though regions and variants use a capital M to denote a region in mitochondrial DNA, rather than a lowercase.

https://gnomad.broadinstitute.org/gene/ENSG00000198899?dataset=gnomad_r4